resources/translations/de.lng

```diff
@@ -2325,6 +2325,9 @@ MSCDEX: Unbekannter Fehler.
 :MSCDEX_WARNING_NO_OPTION
 MSCDEX: Warnung: Ignoriere nicht unterstützte Option '%s'.
 
+.
+:PROGRAM_MOUNT_READONLY
+Laufwerk mit Lesezugriff eingebunden
```

Comment:
The original English string contains an empty line after the message:

```
:PROGRAM_MOUNT_READONLY
Mounted read-only

.
```

Translated string should have an empty line, too. BTW - please do all the changes in a single commit, there is really no need to put such trivial changes into separate commits.